profiles/default/linux/amd64/17.0/musl/make.defaults

```diff
@@ -0,0 +1,10 @@
+# Copyright 1999-2015 Gentoo Foundation.
```

Comment:
Sorry for not noticing this earlier but I think you need to update your calendar ;-).